### DISCUSSION TITLE
PA & GA on the same site

### DISCUSSION THREAD

**Discussion by dmudro** [plausible/analytics#1987] [Anything Else]
while evaluating paid hosted Plausible I wonder if there are any known issues / concerns when including PA and Google Analytics at the same time, i. e. having both PA and GA script tags in the head on the same site?

I do not assume any additional technical issues or conflicts but wonder about any best practice running these side by side for a limited time.

**Comment by metmarkosaric** [comment]
thanks for reaching out @dmudro! there are no issues or conflicts with doing this. most of our subscribers come from GA so they usually test us alongside GA for the trial period to compare the differences etc. 

btw take a look at our proxy if you want to see the largest difference in stats: https://plausible.io/docs/proxy/introduction

**Reply by dmudro** [reply]
great stuff, thanks for a prompt reply. went with the proxy integration and am looking forward to evaluating the product.

**Reply by metmarkosaric** [reply]
sounds like a plan! enjoy the trial!

**Reply by dmudro** [reply]
and we are sold! thanks for your early support. will keep both GA & Plausible for the time being to see if Plausible is as good beyond the trial including the paused import ;) keen to get to know the roadmap better too. looking forward!

**Reply by metmarkosaric** [reply]
Google Analytics import is now back!

https://plausible.io/docs/google-analytics-import

**Reply by lightson** [reply]
Out of interest @metmarkosaric, what happens to the concurrent data during import?

What I mean is... Whilst evaluating PA I had GA still active for several weeks. If I import from GA will I see duplication of data from that period?

**Reply by metmarkosaric** [reply]
HI @lightson! Data is imported from your first Google Analytics visitor until your first Plausible visitor. This is to avoid double-counting visits. You can read more here https://plausible.io/docs/google-analytics-import

**Reply by lightson** [reply]
Apologies, I should have read the page all the way through. That's great news!

**Comment by dmudro** [comment]
hi @metmarkosaric, in our instance Plausible reports 30% greater bounce rate than GA (served via GTM). the number seems quite high to me and I wonder if there is a "typical" delta for other implementations we should expect to see as a rule of thumb.

as for the explanation it could come from unique visitors as Plausible counts double the amount than GA! once again that is quite a lot and hardly attributable to ad blockers etc.

is there a known difference in counting uniques between the two (GA vs Plausible)?

**Comment by RobertJoonas** [comment]
Hi @dmudro!

Due to our privacy-friendly tracking and not using any persistent user identifiers, we cannot track "returning visitors". This means that if one visitor goes to your site on 5 different days, GA would show it as one unique visitor, while we would show it as 5. You can read more about how we count unique visitors here: https://plausible.io/data-policy

I doubt that the difference in unique visitors would result in a 30% difference in bounce rate though. Bounce rate also depends on the custom events you're sending from your site. With a default Plausible snippet, you're only counting pageviews and not any custom events. So if you take the following scenario:

1. A visitor goes to your site
2. A page view event is reported to both Plausible and GA
3. Visitor takes a custom action (e.g. a link click)
4. GA is notified of this custom action, but Plausible is not

Then GA would not count this session as a bounce (as the user engaged), but Plausible would, because it wasn't notified of the custom action.

To explore custom events in Plausible, see: https://plausible.io/docs/custom-event-goals. We also support some custom events (`outbound links` and `file-downloads`) out-of-the-box through script extensions: https://plausible.io/docs/script-extensions

**Reply by dmudro** [reply]
thank you so much. I understand custom events but how come a session with internal link click counts as bounce in Plausible?

isn't that against the bounce definition? i. e. if a user clicks on a link upon visiting our entry page (i. e. homepage) they did not bounce and I'd expect not to add a custom event for a standard click.

**Reply by RobertJoonas** [reply]
Sorry, link click as a custom action was a misleading example. But yes, it shouldn't count as a bounce. An internal link click would mean navigating to a different page on your site, which if tracked by Plausible, would send another pageview event. Consider the same example with "link click" as an "outbound link click" instead.

**Reply by dmudro** [reply]
got it thanks. 

yes it crossed my mind that the problem is not Plausible bounce rate too high (50%) but GA too low (20%). there is no custom event in GA though that would artificially decrease the bounce rate there. 

from a technical standpoint, the property in question is powered by Next.js and given Plausible has support for single page apps, I do not assume client-side routing would cause the bounce rate too high. i. e. I expect client side route change equal to page view meaning click throughs within the same site (but client side history without HTML refresh) do not increase bounce rate in Plausible. can you confirm please?

**Reply by RobertJoonas** [reply]
A 50% bounce rate sounds pretty healthy to me, whereas I'm not sure about 20% on a single page app.

We do have the `script.hash.js` extension to support SPA-s. When using it, every client side route change will send a pageview event, so I can confirm that the bounce rate should not increase.

What about the ratio of visitors/pageviews? Notice any significant differences there?
